package/com.unity.formats.usd/Runtime/Scripts/IO/Scene/ExportHelpers.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.IO;
+using UnityEditor;
+using UnityEngine;
+using USD.NET;
+
+namespace Unity.Formats.USD
+{
+    public static class ExportHelpers
+    {
+#if UNITY_EDITOR
+        public static Scene InitForSave(string defaultName, string fileExtension = "usd,usda,usdc")
+        {
+            var filePath = EditorUtility.SaveFilePanel("Export USD File", "", defaultName, fileExtension);
```

Comment:
I would split this function differently:
The only editor only function is EditorUtility.SaveFilePanel: add that in the editor assembly and keep the rest of of this function as a runtime function.

Reply:
Fixed in 222b79c9